core/trashcan.ts

```diff
@@ -697,6 +697,14 @@ export class Trashcan
     };
     return blockInfo;
   }
+
+  wouldDelete(_element: IDraggable): boolean {
+    return false;
```

Comment:
is this supposed to be part of this change? shouldn't the traschcan be a delete area?

Reply:
Ah whoops yes! This was just for testing and accidentally got committed. Thank you for the catch.